src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -132,6 +132,13 @@ public async Task<IEnumerable<Type>> GetExtensionsStartupTypesAsync()
             }
             else
             {
+                string systemPath = Path.Combine(_rootScriptPath, ScriptConstants.AzureFunctionsSystemDirectoryName);
```

Comment:
No need to execute this line if `isDotnetIsolatedApp` is false. You may inline this in line 137.

Reply:
I need this path to check if the directory exists in the next line. Are you suggesting to do something else?

Reply:
I think this is what @kshyju is recommending-

`if (isDotnetIsolatedApp && 
    !SystemEnvironment.Instance.IsPlaceholderModeEnabled() && !Directory.Exists(Path.Combine(_rootScriptPath, ScriptConstants.AzureFunctionsSystemDirectoryName)))
{
    _logger.NoAzureFunctionsFolder();
}`